metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/database/SqlxmlJdbcTemplate.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.database;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import java.sql.SQLXML;
+import java.util.List;
+
+public class SqlxmlJdbcTemplate {
```

Comment:
SqlXmlJdbcTemplate ?

Reply:
If the clarification is on name, we can use this class to get any XML file from other tables apart from Analysis. Hence named it as generic SqlXmlJdbcTemplate

Reply:
Since the package is java.sql.```SQLXML``` i took it as single word. We could change to SqlXml aswell. Let me know.